src/java.desktop/macosx/native/libawt_lwawt/awt/AWTWindow.m

```diff
@@ -999,7 +999,7 @@ - (void)windowDidExitFullScreen:(NSNotification *)notification {
 }
 
 - (void)sendEvent:(NSEvent *)event {
-        if ([event type] == NSLeftMouseDown || [event type] == NSRightMouseDown || [event type] == NSOtherMouseDown) {
```

Comment:
since you are touching this, maybe we can format it for better legibility !
```
if ([event type] == NSEventTypeLeftMouseDown ||
     [event type] == NSEventTypeRightMouseDown ||
     [event type] == NSEventTypeOtherMouseDown))
{
```